### PR TITLE
Adding Jenkins File

### PR DESCRIPTION
**Pull request by venkayya**

### CODE REVIEW

**Comment by venkayya** [comment]
I signed it!

**Comment by chrislovecnm** [comment]
@venkayya do not worry about the CLA bot. Also you need to ammend you commit and change the author. Also please squash your commits.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          checkout scm
+
+          // initialize submodule and update
+          sh "git submodule update --init --recursive"
+
+          // Setup gcloud service account access
+          sh "gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}"
+          sh "gcloud config set compute/zone ${env.ZONE}"
+          sh "gcloud config set core/project ${env.PROJECT_ID}"
+          sh "gcloud config set compute/region ${env.REGION}"
```
Comment (chrislovecnm) [inline Jenkinsfile]:
```
          sh "git submodule update --init --recursive"

          // Setup gcloud service account access
          sh "gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}"
          sh "gcloud config set compute/zone ${env.ZONE}"
          sh "gcloud config set core/project ${env.PROJECT_ID}"
          sh "gcloud config set compute/region ${env.REGION}"
```

We do not require these command.  please update makefile.  @donovanjamesl can help you getting with getting Jenkins running.

Comment (venkayya) [reply Jenkinsfile]: Removed submodule section from jenkinsfile
updated makefile with target lint
created job in ci

**Comment by emailbob** [comment]
test this please

**Comment by venkayya** [comment]
test this

**Comment by chrislovecnm** [comment]
Testing this please

**Pull request merged**